src/foundry/client/head.d.ts

```diff
@@ -1,3 +1,4 @@
+import * as ProseMirrorObject from '../prosemirror/prosemirror';
 export {};
```

Comment:
This can now be removed, that we have an import here. It was just to force this to be a module.